package.json

```diff
@@ -9,10 +9,11 @@
     "src"
```

Comment:
Rebase with master to resolve merge conflicts with `package.json` and `package-lock.json`